terraform/grafana-alerts/alert-rules.tf

```diff
@@ -0,0 +1,154 @@
+resource "grafana_rule_group" "oracle_relayers" {
+  name             = "Oracle Relayer Alerts"
+  folder_uid       = var.oracle_relayers_folder.uid
+  interval_seconds = 60
+
+  dynamic "rule" {
+    for_each = local.chains
+
+    content {
+      name      = "Oldest Report Expired Alert [${title(rule.value)}]"
+      condition = "isExpired"
+      for       = "1m"
```

Comment:
is this the "pending period" in grafana? aka will this alert trigger if the condition is met for a minute? If so maybe it would be better to make this ~5 minutes so it doesn't trigger immediately if for example chainlink has some minimal downtime

Reply:
yeah, i think 5m was the default, I reduced this for faster iteration during testing, will revert to 5mins.

> `for`: The amount of time for which the rule must be breached for the rule to be considered to be Firing. Before this time has elapsed, the rule is only considered to be Pending. Defaults to 0.